### PR TITLE
OWSave is used to save images and graphs within imageViewer and TreeV…

### PR DESCRIPTION
**Pull request by VesnaT**
…iewer

### CODE REVIEW

File: Orange/data/io.py [hunk 1]
```diff
@@ -656,3 +669,51 @@ def read_file(cls, file, _=None):
     def write_file(cls, filename, table):
         pickle.dump(table, open(filename, "wb"))
 
+    def write(self, filename, table):
+        self.write_file(filename, table)
+
+
+@FileFormats.register("Portable Network Graphics", ".png")
+class PngFormat:
```
Comment (kernc) [inline Orange/data/io.py]: Consider also [QSvgGenerator](http://doc.qt.io/qt-4.8/qsvggenerator.html#details)?

File: Orange/data/io.py [hunk 2]
```diff
@@ -5,13 +5,14 @@
 from itertools import chain
 
 import bottlechest as bn
-import numpy as np
```
Comment (kernc) [inline Orange/data/io.py]: Isn't `np` used further below in the file?

File: Orange/data/io.py [hunk 3]
```diff
@@ -5,13 +5,14 @@
...
+from sklearn import tree
 from scipy import sparse
 # We are not loading openpyxl here since it takes some time
 
 from Orange.data import _io
```
Comment (kernc) [inline Orange/data/io.py]: Seeing where `_io` is used, wouldn't it make sense to just move the exotic import into that function? For a faster boot time at the least! :smiley:

File: Orange/data/io.py [hunk 4]
```diff
@@ -6,12 +6,13 @@
 
 import bottlechest as bn
 import numpy as np
+from sklearn import tree
```
Comment (kernc) [inline Orange/data/io.py]: Try moving this import below all the others, or where it's only ever used?

**Comment by kernc** [comment]
... When you get it working, some tests would be in order. :]

**Comment by astaric** [comment]
The tests fails, because tests for pull requests did not install pyqt on travis (installation used to be slow). If you rebase on current master, the test will probably pass.

Do we really want to have PyQt imports inside the Orange.**data**.io? As FileFormats is a registry, readers and writers can be implemented anywhere. If ImgFormat and classes that extend it can only save scenes, that are usually related to widgets, perhaps a more suitable place for this code would be inside Orange.widgets package?

**Comment by lanzagar** [comment]
Try to save a png from the Calibration Plot widget.
I get a 'module' not callable error
Problematic lines:
https://github.com/biolab/orange3/pull/499/files#diff-8c9565c33ed8859fe616fb9774235f27R33
https://github.com/biolab/orange3/pull/499/files#diff-8c9565c33ed8859fe616fb9774235f27R74

Check for similar problem in SVG...

**Comment by kernc** [comment]
Pull requests, as contained wholes, should (imho) not be merged until [properly squashed](https://www.google.com/search?q=github+squash).

**Pull request merged**